### PR TITLE
feat/106: converting playht speed option to float instead of integer

### PR DESCRIPTION
**Pull request by rammohan-y**
converting playht speed option to float instead of integer

### CODE REVIEW

File: mod_playht_tts/playht_glue.cpp [hunk 1]
```diff
@@ -774,7 +774,7 @@ extern "C" {
       cJSON_AddStringToObject(jResult, "quality", p->quality);
     }
     if (p->speed) {
-      cJSON_AddNumberToObject(jResult, "speed", atoi(p->speed));
+      cJSON_AddNumberToObject(jResult, "speed", atof(p->speed));
```
Comment (xquanluu) [inline mod_playht_tts/playht_glue.cpp:777]: could you help to update this:
https://github.com/jambonz/freeswitch-modules/blob/main/mod_dub/tts_vendor_parser.cpp#L381 for md_dub

Comment (davehorton) [inline mod_playht_tts/playht_glue.cpp:777]: Its possible the user may provide a string that is neither a float nor an integer.  I would like to discard it in this case.  One way to do it is like so
```
double val = strtod(p->speed, NULL);
if (val != 0.0) {
   cJSON_AddNumberToObject(jResult, "speed", val);
}
```
Can you make that change and test it, and test with a garbage value as well.

**Review by davehorton — CHANGES_REQUESTED** [review]

**Review by davehorton — APPROVED** [review]

**Comment by davehorton** [comment]
@rammohan-y you must sign off on **all** commits in this repo before they can be accepted.  Please review [these guidelines](https://github.com/jambonz/freeswitch-modules/blob/main/CONTRIBUTING.md) and if you agree, push a commit with the `-s` flag to indicate signoff.

**Review by davehorton — CHANGES_REQUESTED** [review]
signoff required

**Comment by rammohan-y** [comment]
> signoff required

How do i signoff?

**Comment by rammohan-y** [comment]
> Quote reply

signed

**Review by davehorton — APPROVED** [review]

**Pull request merged**